Python/71_Trust_But_Verify.ipynb

```diff
@@ -80,7 +80,7 @@
     "%run update_path_to_download_script\n",
     "from downloaddata import fetch_data as fdata\n",
     "\n",
-    "OUTPUT_DIR = 'Output'\n",
+    "OUTPUT_DIR = './Output'\n",
```

Comment:
Why was this modified?

Reply:
From what I could tell, Windows was handling the path without ./ as an absolute.  Adding the ./ forced it to be relative.

Reply:
I don't think this is the issue, as the same name is used in all the notebooks and I assume those passed the testing on windows. Not sure what is different in this notebook.

Reply:
Ah, you're right. I just retested this, and it works fine in the original form.  I must have gotten it mixed it up with other issues.  I'll revert this.